service/scaladsl/kafka/server/src/test/scala/com/lightbend/lagom/scaladsl/kafka/broker/ScaladslKafkaApiSpec.scala

```diff
@@ -77,17 +77,20 @@ class ScaladslKafkaApiSpec
       override lazy val lagomServer            = serverFor[TestService](new TestServiceImpl)
 
       override def additionalConfiguration = {
-        import scala.collection.JavaConverters._
-        super.additionalConfiguration ++ ConfigFactory.parseMap(
-          Map(
-            "akka.remote.artery.canonical.port"             -> "0",
-            "akka.remote.artery.canonical.hostname"         -> "127.0.0.1",
-            "akka.persistence.journal.plugin"               -> "akka.persistence.journal.inmem",
-            "akka.persistence.snapshot-store.plugin"        -> "akka.persistence.snapshot-store.local",
-            "lagom.cluster.exit-jvm-when-system-terminated" -> "off",
-            "lagom.cluster.bootstrap.enabled"               -> "off",
-            "lagom.services.kafka_native"                   -> s"tcp://localhost:$kafkaPort"
-          ).asJava
+        super.additionalConfiguration ++ ConfigFactory.parseString(
+          s"""
+            akka.remote.artery.canonical.port = 0,
+            akka.remote.artery.canonical.hostname = 127.0.0.1
+            akka.persistence.journal.plugin = akka.persistence.journal.inmem
+            akka.persistence.snapshot-store.plugin = akka.persistence.snapshot-store.local
+            lagom.cluster.exit-jvm-when-system-terminated = off
+            lagom.cluster.bootstrap.enabled = off
+            akka.discovery.method = config
```

Comment:
This changes the default `method`  for discovery of all the Actor System. This could have an impact on other areas (?).

Reply:
No. It's settings only for this concreate test. For the customers, we set `akka.discovery.method` in module `lagom-akka-discovery-service-locator-core`. But the classpath of this test does not contain this module.